pkg/cmd/cli/cmd/importer/dockercompose.go

```diff
@@ -34,7 +34,9 @@ During transformation fields in the compose syntax that are not relevant when ru
 a containerized platform will be ignored and a warning printed.
 
 The command will create objects unless you pass the -o yaml or --as-template flags to generate a
-configuration file for later use.`
+configuration file for later use.
+
+This command is experimental and the behavior is subject to change.`
```

Comment:
`Experimental: This command is under active development and may change without notice` probably